clang/test/CodeGenCXX/dependent-template-type-scope.cpp

```diff
@@ -0,0 +1,18 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -emit-llvm -debug-info-kind=standalone  -o - %s | FileCheck %s
+
+struct X {
```

Comment:
You should either add some CHECK lines for 'X' or just remove it.  At present, it isn't relevant to the test.